examples/default/channels/inventory.events/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: inventory.events
+name: Inventory Events
+version: 1.0.1
+summary: |
+  The topic on which measured values may be produced and consumed.
+owners:
+    - dboyne
+address: smartylighting/streetlights/1/0/event/5/lighting/measured
```

Comment:
also i recommend look at specs and see if any thought naming can be found 

a channel often must have a host , but here as we talk abounts events inside a channel markdown maybe this can be rethinked

Reply:
Hey @XaaXaaX thanks for the comment.

I'm not sure I 100% understand, any chance you could explain some more with maybe an example? Thanks mate!

Reply:
Oh @boyney123 
Just i tried to say do we title the attribute address or host?
Just i m trying to adopt the same nomination provided by standard specifications 
But practically, i assume a channel can not be pointed only by a host

Reply:
Ahhh OK cool, yeah I took address from https://www.asyncapi.com/docs/reference/specification/v3.0.0#channelObject, and protocol from the same place, yeah anywhere we can stear towards standards I'm always down for!